ocaml/xapi/vusbops.ml

```diff
@@ -0,0 +1,63 @@
+(*
+ * Copyright (C) Citrix Systems Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation; version 2.1 only. with the special
+ * exception on linking described in file LICENSE.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *)
+module D=Debug.Make(struct let name="vusbops" end)
+open D
+
+open Stdext
+open Listext
+
+type vusb = {
+  vusb_ref: API.ref_VUSB;
+  usb_group_ref: API.ref_USB_group;
+  other_config: (string * string) list;
```

Comment:
This isn't being used, and the name clashes with lots of other other_configs, so let's remove it.

Reply:
Ok, make sense.

Reply:
updated.